src/components/Element.js

```diff
@@ -40,11 +38,11 @@ const Element = (type: string, hocOptions: {sourceType?: string} = {}) =>
     };
     static defaultProps = {
       className: '',
-      elementRef: noop,
-      onChange: noop,
-      onBlur: noop,
-      onFocus: noop,
-      onReady: noop,
+      elementRef: undefined,
```

Comment:
We don't need to include these in `defaultProps` anymore!

Reply:
defaultProps